sagemaker_processing/fairness_and_explainability/fairness_and_explainability.ipynb

```diff
@@ -389,9 +415,9 @@
    "source": [
     "#### Writing BiasConfig\n",
     "SageMaker Clarify also needs information on what the sensitive columns (`facets`) are, what the sensitive features (`facet_values_or_threshold`) may be, and what the desirable outcomes are (`label_values_or_threshold`).\n",
-    "Clarify can handle both categorical and continuous data for `facet_values_or_threshold` and for `label_values_or_threshold`. In this case we are using categorical data.\n",
+    "SageMaker Clarify can handle both categorical and continuous data for `facet_values_or_threshold` and for `label_values_or_threshold`. In this case we are using categorical data.\n",
     "\n",
-    "We specify this information in the `BiasConfig` API. Here that the positive outcome is earning >$50,000, Sex is a sensitive category, and Female respondents are the sensitive group."
+    "We specify this information in the `BiasConfig` API. Here that the positive outcome is earning >$50,000, Sex is a sensitive category, and Female respondents are the sensitive group. `group_name` is used to form subgroups for the measurement of Conditional Demographic Disparity in Labels (CDDL) and Conditional Demographic Disparity in Predicted Labels (CDDPL) with regards to Simpson’s paradox."
```

Comment:
maybe add which value corresponds to in the dataset (0?) since Sex is a binary category in this dataset.

Reply:
Thanks. It is mentioned in line 256
>    "A quick note about our encoding: the \"Female\" Sex value has been encoded as 0 and \"Male\" as 1."